content/en/serverless/google_cloud_run/_index.md

```diff
@@ -189,112 +177,95 @@ ENV DD_TRACE_PROPAGATION_STYLE=datadog
 
 {{< /programming-lang >}}
-{{< programming-lang lang="nodejs" >}}
+{{< programming-lang lang="php" >}}
 
-Follow [these instructions][2] to install and configure the Node tracing library in your application to capture and submit traces. 
+Instrument your application with the Datadog Agent by adding the following lines to your Dockerfile. You may need to adjust these examples depending on your existing Dockerfile setup.
 
-[Sample code for a simple Node.js application][1].
+```
+# copy the Datadog `serverless-init` into your Docker image
+COPY --from=datadog/serverless-init /datadog-init /app/datadog-init
 
-[1]: https://github.com/DataDog/crpb/tree/main/js
-[2]: /tracing/trace_collection/dd_libraries/nodejs/?tab=containers#instrument-your-application
+# Copy and install the Datadog PHP Tracer
+ADD https://github.com/DataDog/dd-trace-php/releases/latest/download/datadog-setup.php /datadog-setup.php
+RUN php /datadog-setup.php --php-bin=all
 
-{{< /programming-lang >}}
-{{< programming-lang lang="java" >}}
+# optionally add Datadog tags
+ENV DD_SERVICE=datadog-demo-run-ruby
+ENV DD_ENV=datadog-demo
+ENV DD_VERSION=1
 
-Follow [these instructions][2] to install and configure the Java tracing library in your application to capture and submit traces. 
+# change the entrypoint to wrap your application into the Datadog serverless-init process
+ENTRYPOINT ["/app/datadog-init"]
 
-[Sample code for a simple Java application][1].
+# use the following for an apache and mod_php based image
+RUN sed -i "s/Listen 80/Listen 8080/" /etc/apache2/ports.conf 
+EXPOSE 8080
+CMD ["apache2-foreground"]
 
-[1]: https://github.com/DataDog/crpb/tree/main/java
-[2]: /tracing/trace_collection/dd_libraries/java/?tab=containers#instrument-your-application
+# use the following for an nginx and php-fpm based image
+RUN ln -sf /dev/stdout /var/log/nginx/access.log && ln -sf /dev/stderr /var/log/nginx/error.log
+EXPOSE 8080
+CMD php-fpm; nginx -g daemon off;
+```
 
 {{< /programming-lang >}}
-{{< programming-lang lang="dotnet" >}}
+{{< /programming-lang-wrapper >}}
 
-#### 
+#### Install Agent with buildpack
 
-Follow the instructions to install and configure the [.NET Core tracing library][1] and the [.NET Framework tracing library][2]. 
+[`Pack Buildpacks`][3] provide a convenient way to package your container without using a Dockerfile. This example uses the Google Cloud container registry and Datadog serverless buildpack.
 
-[1]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/dotnet-core?tab=containers#custom-instrumentation
-[2]: /tracing/trace_collection/dd_libraries/dotnet-framework/?tab=containers#custom-instrumentation
+**Note**: Install the tracing library for your language before running the buildpack. If you're using a Dockerfile, you can skip this step.
 
-{{< /programming-lang >}}
-{{< programming-lang lang="ruby" >}}
+[Go library][1]
+[Python library][2]
+[NodeJS library][3]
+[Java library][4]
+[Ruby library][5]
+[PHP Library][6]
```

Comment:
We're going to want to change this to something more reasonable. Ideally, we just want this for the Buildpack instructions, as we don't need it for Dockerfile anymore.